.github/workflows/multi-arch-test-build.yml

```diff
@@ -182,44 +182,58 @@ jobs:
 
```

Comment:
This is already superseeded by https://github.com/openwrt/actions-shared-workflows/pull/39